src/reference/asciidoc/index.adoc

```diff
@@ -41,3 +41,6 @@ include::si-amqp.adoc[]
 In addition to this reference documentation, there exist a number of other resources that may help you learn about AMQP.
 
 include::further-reading.adoc[]
+
+[appendix]
+include::appendix.adoc[]
```

Comment:
Not sure why but in the PDF we have this:
![manual](https://cloud.githubusercontent.com/assets/1071590/13014732/55e57518-d183-11e5-8002-ff3ce9f943ac.jpg)
Looks like in the `index.adoc` we should place new page breaker `<<<` before `include::whats-new.adoc[]`